dub.sdl

```diff
@@ -36,8 +36,13 @@ subPackage {
 		auto destFile = file.replace(srcDir, destDir);
 		destFile.dirName.mkdirRecurse;
 		file.readText.replace("std.experimental.allocator", "stdx.allocator")
+			.replace("std.conv", "stdx.allocator.conv")
 		 	.toFile(destFile);
 	}
+	pkgDir.buildPath("std", "conv.d")
+		  .readText
+		  .replace("std.conv", "stdx.allocator.conv")
+		  .toFile(destDir.buildPath("conv.d"));
```

Comment:
I'm surprised that this works, and I think this can be quite brittle in the future - e.g. `emplace` may depend on a fix or new feature in some other module. But given that we have a CI to check for such issues, I'd say we're fine for this experimental feature. But these sorts of hacks are not going to be sustainable, so sooner or later we should look into a more future-proof solution.